docs/versions.rst

```diff
@@ -4,6 +4,20 @@ Version History
 
 .. automodule:: more_itertools
 
+4.3.0
+-----
+
+* New itertools:
+    * :func:`last` (thanks to tmshn)
+    * :func:`replace` (thanks to pylang)
```

Comment:
you're missing bbayles ;)

Reply:
I've just been doing hat tips for community contributions. I'm in for the additions from before I was maintainer.